### PR TITLE
Adding content from the Nordic Pandemic Infrastructure (NeIC PaRI) project

### PR DESCRIPTION
**Pull request by wna-se**
This pull request aims to contribute content from the Nordic Pandemic Infrastructure (NeIC PaRI) under the relevant parts of the IDTkit (#32). So far I’ve only added a markdown version of the guide under Showcase and I open this pull request to keep track of the effort and to open up a discussion on where different parts of the content could go.

### CODE REVIEW

**Comment by bgruening** [comment]
Great work @wna-se

**Review by LianeHughes — COMMENTED** [review]
I started reading this through (looks really good!), but didn't continue to read through thoroughly as I started to wonder a bit more about where this should go. Right now, it's added as a 'showcase'. I see the logic in placing it as a showcase because it's a project that brought together multiple technologies. However, when we defined 'showcases', I thought that they would be more e.g. the COVID-19 Data Portal/Platform, i.e. tools. Under the strictest definition of 'tool', this would not really fit as a showcase. However, I think that I would be useful to keep it all together, and maybe a showcase is the best way to do that. I think it might also be good to have this information at least linked in the 'pathogen characterisation' sections though. Having it linked at various sections (rather than actually split between sections), would perhaps make maintenance of this content easier than breaking it all up. I think it depends on out definition of 'what is a showcase' though, for where it might fit best

**Comment by rabuono** [comment]
Thank you @wna-se for the contribution and all the time you already spent on this!

I do get divided between keeping all the content together in a single showcase to represent the entire product together, and splitting the general/universal advice and guidelines from the showcase specific application.

I do fear that keeping all the info together will lead to a lot of content duplication and that one of the nice products out of IDTk will be a distilled set of best practices out of all the showcases.

File: showcase/neic_pari.md [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+- [ELIXIR Norway Research support](https://elixir.no/helpdesk)
+- [ELIXIR Norway COVID-19 resources](https://elixir.no/corona)
+- [ELIXIR Norway SARS-Cov-2 database](https://covid19.sfb.uit.no/)
+
+### Sweden
+
+- [Swedish COVID-19 Data Portal Support Services](https://www.covid19dataportal.se/support_services/)
+- [SciLifeLab Data Guidelines](https://scilifelab-data-guidelines.readthedocs.io/en/latest/docs/covid-19/index.html)
+
+### NeIC PaRI case study
```
Comment (rabuono) [inline showcase/neic_pari.md:342]: Would the three cases make for three independent showcase pages?
They do seem to fit as combinations of tools, services, and standards to gather, process, and submit data.

Comment (LianeHughes) [reply showcase/neic_pari.md:342]: I believe that the three below could be showcases too, perhaps and maybe linked here when they're up? Maybe the national pages could also be linked for the above examples too?

File: showcase/neic_pari.md [hunk 2]
```diff
@@ -0,0 +1,351 @@
+---
+title: The NeIC PaRI partners’ guide to SARS-CoV-2 genome data sharing
+description: 
+---
+Produced by the Nordic Pandemic Research Infrastructure project (NeIC PaRI)
+[https://neic.no/pari/](https://neic.no/pari/)
+
+This document aims to 1) outline what constitutes a good and ultimately (re)usable viral genome data record in a data repository with a Nordic perspective; and 2) provide guidance to projects, labs and other organisations producing or commissioning viral sequencing data.
+
+## Introduction
```
Comment (LianeHughes) [inline showcase/neic_pari.md:10]: SPLIT SUGGESTION: Could generalise this to be more of a description about data description in the pathogen characterisation section

File: showcase/neic_pari.md [hunk 3]
```diff
@@ -0,0 +1,351 @@
...
+
+Combining information on virus characteristics with clinical and epidemiological data is desirable when studying various aspects of a pandemic. Viral genomic data provide insight into some important virus characteristics and should be reported to and shared with the global research and clinical communities as early and as openly as possible. The following text will provide guidance on sharing data from viral genome sequencing. The focus will be on the processing and documentation required for sharing the data files containing reads produced by sequencing instruments and consensus sequences produced by analytic workflows for genome assembly—both of which constitute valuable resources for different audiences.
+
```
Comment (LianeHughes) [inline showcase/neic_pari.md:13]: 1. END

File: showcase/neic_pari.md (hunk 1 above)
Comment (LianeHughes) [inline showcase/neic_pari.md:342]: 6 cont. SPLIT SUGGESTION Perhaps the 'case study' could also integrate into national pages?

File: showcase/neic_pari.md (hunk 2 above)
Comment (LianeHughes) [inline showcase/neic_pari.md:10]: 1. SPLIT SUGGESTION: Could generalise this to be more of an introduction about data description in the pathogen characterisation section

File: showcase/neic_pari.md (hunk 3 above)
Comment (LianeHughes) [inline showcase/neic_pari.md:13]: 1. END SPLIT SUGGESTION

**Pull request closed without merging**